translator/isValid.go

```diff
@@ -94,5 +94,5 @@ func IsValidRetentionDays(days int) bool {
 }
 
 func IsValidLogGroupClass(class string) bool {
-	return slices.Contains(ValidLogGroupClasses, class)
+	return slices.Contains(ValidLogGroupClasses, class) || class == ""
```

Comment:
We might not need this change as this returns false for class =="" which is handled in else section setting the returnVal to ""

Reply:
After discussion, determined that it should not return errors, will change log level